src/gui/advancedsettings.cpp

```diff
@@ -376,7 +379,13 @@ void AdvancedSettings::loadAdvancedSettings()
     m_spinBoxAsyncIOThreads.setValue(session->asyncIOThreads());
     addRow(ASYNC_IO_THREADS, (tr("Asynchronous I/O threads") + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#aio_threads", "(?)"))
             , &m_spinBoxAsyncIOThreads);
-
+    // stop tracker timeout
+    m_spinBoxStopTrackerTimeout.setMinimum(1);
+    m_spinBoxStopTrackerTimeout.setMaximum(30);
```

Comment:
I would rather not impose an upper limit. So either remove this line (the default limit is 99 IIRC) or make it `std::numeric_limits<int>::max()`.